PublishParasoftTestResults/publishtestresults.ts

```diff
@@ -269,9 +279,44 @@ function checkStaticAnalysisViolations(sarifReports: string[], index: number) {
     }
 }
 
+function loadSettings(localSettingsPath : string) {
+    if (isNullOrWhitespace(localSettingsPath)) {
+        tl.warning('Local settings file path is not specified.');
+        return null;
+    }
+
+    let localSettingsFile = tl.resolve(tl.getVariable('System.DefaultWorkingDirectory'), localSettingsPath);
+    tl.debug('Path to local settings is ' + localSettingsFile);
+
+    return loadProperties(localSettingsFile);
+}
+
+function loadProperties(localSettingsFile : string) {
+    let input: string;
+    try {
+        input = fs.readFileSync(localSettingsFile, 'utf-8');
+    } catch (err) {
+        tl.error('Error while reading local settings file.');
+        return null;
+    }
+
+    try {
+        let props = dp.parse(input, false);
+        if (typeof props === 'undefined' || props === null || 
```

Comment:
From the source code of the library I can see we only need to check "props" as empty object. I think the only condition check we can keep is: 
!props || Object.keys(props).length === 0
!props is just to be safe.
![image](https://user-images.githubusercontent.com/35082985/230508980-4b17fef6-9274-4ae2-bfa5-b7faa9e7e158.png)